### ISSUE TITLE
Cannot copy&paste Linting error messages on Mac

### ISSUE THREAD

**Issue by MaxTru** [camunda/camunda-modeler#2716]
__Describe the Bug__

Cannot copy&paste Linting error messages on Mac

__Steps to Reproduce__

1. Be on MacOS
2. Create Cloud BPMN
3. Create plain task
4. Open error view
5. Select the error message
6. Try to copy it via keyboard or right-click
=> Clipboard remains empty

__Expected Behavior__

On MacOS:

- [ ] I can copy via keyboard what I selected

Stretch goal:

- [ ] I can right-click and copy on all OS

__Environment__

- OS: MacOS
- Camunda Modeler Version: 5.0.0-alpha.0

raised by @meyerdan via https://camunda.slack.com/archives/CKGH9LR40/p1643373696260759

**Comment by MaxTru** [comment]
@barmac or @pinussilvestrus could one of you please reproduce on MacOS

**Comment by andreasgeier** [comment]
On my Mac, I can select parts of the message and copy/paste it. But there is no visible selection of the error label and the activity ID and the pasted text does not contain everything.

![Screen Shot 2022-01-28 at 14 39 03](https://user-images.githubusercontent.com/938244/151556476-f6accd7f-72f3-41b5-b999-528bf0d255c4.png)

Pasted on GitHub:

Element of type <bpmn:Task> not supported by Camunda Cloud 1.3
Element of type <bpmn:Task> not supported by Camunda Cloud 1.3
Element of type <bpmn:Task> not supported by Camunda Cloud 1.3

Pasted on Mac Notes:

Element of type <bpmn:Task> not supported by Camunda Cloud 1.3
Error : Activity_1ug5q3z - Element of type <bpmn:Task> not supported by Camunda Cloud 1.3
Error : Activity_19iu790 - Element of type <bpmn:Task> not supported by Camunda Cloud 1.3

## Camunda Modeler system information
 * Version: 5.0.0-alpha.0
 * Operating System: Mac OS 12.1.0 
 * Plugins: Cloud Connect
 * Execution Platform: BPMN - Camunda Cloud

**Comment by andreasgeier** [comment]
cf. https://github.com/camunda/camunda-modeler/issues/2697

**Comment by pinussilvestrus** [comment]
For it sometimes does not work when the actions are not enabled in the edit menu. We had a similar issue for the Log panel in the past: https://github.com/camunda/camunda-modeler/issues/1380#issuecomment-597670186

**Comment by meyerdan** [comment]
It does not work for me on my mac, here is my system info

## Camunda Modeler system information
 * Version: 5.0.0-alpha.0
 * Operating System: Mac OS 11.5.0 
 * Plugins: Cloud Connect
 * Execution Platform: BPMN - Camunda Cloud

**Comment by nikku** [comment]
Should this not be a duplicate of https://github.com/camunda/camunda-modeler/issues/2687 and already fixed?

**Comment by MaxTru** [comment]
> Should this not be a duplicate of #2687 and already fixed?

No, not really. #2687 was fixed by making it possible to select text. Now this works for @meyerdan , but I guess the keyboard copy does not work in some situations (see https://github.com/camunda/camunda-modeler/issues/2716#issuecomment-1024238471)

**Comment by MaxTru** [comment]
Marking for next milestone, since we should fix this for 5.0 stable.
